dashboard/src/components/InfoCard/InfoCard.scss

```diff
@@ -42,33 +42,27 @@
 }
 
 .card-block {
-  overflow: hidden;
-  height: 4.5rem;
-
   .info-card-block {
     display: flex;
 
     .card-icon {
       display: flex;
       width: 64px;
-      height: 64px;
       align-items: center;
       text-align: center;
 
       img {
         width: 64px;
-        max-height: 64px;
       }
     }
 
     .card-description-wrapper {
       display: grid;
       width: 100%;
-      height: 64px;
+      min-height: 7lh;
```

Comment:
After the changes in the clr-ui, some cards were being rendered odly. Changes in this file are for fixing it:

![3](https://github.com/vmware-tanzu/kubeapps/assets/11535726/bf7d616c-8246-46b6-a077-5fad9778d8ce)